.expeditor/verify.pipeline.yml

```diff
@@ -432,6 +432,8 @@ steps:
         limit: 1
 
   - label: "[build] :linux: hab"
+    env:
+      HAB_STUDIO_SECRET_HAB_LICENSE: "accept-no-persist"
```

Comment:
I think `HAB_LICENSE` should be one of the variables allowed to pass the studio boundary. Otherwise every time a user wants to run a studio or build a package, they'll have to set `HAB_STUDIO_SECRET_HAB_LICENSE`. 

Ideally (if possible) the cli would translate any form of license acceptance on the system where it's running into `HAB_LICENSE=value` inside the studio. If not, the user will potentially have to accept the license in every studio or set the appropriate variable in their environment even if they have a license acceptance file persisted on disk.

Reply:
Yeah this is a good point.  I'll spend some time with this.

Reply:
@smacfarlane With my latest changes, all the studios (Linux chroot, Windows, Docker, rootless) _should_ inherit the license settings from the host, either via an explicitly set env var, or by detecting the license file in the correct place.  I'm interested in your feedback on the latest changes.